applications/nrf5340_audio/src/modules/hw_codec.c

```diff
@@ -478,10 +478,105 @@ static int cmd_input(const struct shell *shell, size_t argc, char **argv)
 	return 0;
```

Comment:
Discuss: This file has a generic name, but operations within are sometimes CS47L63 specific.

Reply:
I guess that is because we have only supported the CS47L63 until now